src/pages/components/code-snippet/accessibility.mdx

```diff
@@ -19,79 +19,83 @@ import {
+<Row>
+<Column colLg={8}>
+
+![inline code snippet keyboard interaction](images/code-snippet-accessibility-1.png)
+
+</Column>
+</Row>
+
+By default, each inline code snippet is reachable by Tab and copied with Space or Enter.
+
+<Row>
+<Column colLg={8}>
+
+![single line code snippet interaction, with 2 tab stops](images/code-snippet-accessibility-2.png)
+
+</Column>
+</Row>
+
+The single line code snippet tabstop supports left and right arrow key scrolling.
+
+<Row>
+<Column colLg={8}>
+
+![multi-line code snippet keyboard interaction](images/code-snippet-accessibility-3.png)
 
-Automated, manual and screen reader accessibility verification test has been
-performed on the code snippet React Carbon component.
-[WCAG 2.1 Level A and AA success criteria](https://www.w3.org/TR/WCAG21/) issues
-have been identified and the list of open accessibility violations is available
-in the Carbon Component GitHub repository.
+</Column>
+</Row>
+
+The multi-line’s buttons are reachable by Tab and activated with Space or Enter.
+
+### Labelling and updates
+
+Carbon provides the copy button's default label and tooltip behavior. Carbon
+handles notices about the success of the copy function, as well as updates to the
+Show more mechanism.
+
+<Row>
+<Column colLg={8}>
+
+![hovering on the button exposes ‘copy to clipboard’ tooltip](images/code-snippet-accessibility-4.png)
 
-### Automated test
+</Column>
+</Row>
+
+The code snippet’s buttons expose their labels on hover or focus.
 
 <Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListHead>
-        <StructuredListRow head>
-          <StructuredListCell head>
-            Automated test environment
-          </StructuredListCell>
-          <StructuredListCell head>Results</StructuredListCell>
-        </StructuredListRow>
-      </StructuredListHead>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - macOS Mojave version 10.14.6 with VoiceOver
-            <br />
-            - Chrome version 77.0.3865.90
-            <br />
-            - Dynamic Assessment Plugin (DAP) version 1.8.0.0 - IBM
-            Accessibility WCAG 2.1 Sept. 2019 Ruleset
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>DAP test:</strong>
-            <br />- Violations
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![the activated button shows a ‘copied!’ message](images/code-snippet-accessibility-5.png)
+
+</Column>
 </Row>
+
+The results of activating buttons are provided in text.
+
+## Developer considerations
+
+Keep this in mind if you’re modifying Carbon or creating a custom component:
+– the inline code text is implemented as a button so its text can be copied
+– single line snippets take an additional tabstop to support arrow key scrolling
```

Comment:
I think these need an empty line above them and a simple dash to get the proper list styling. Right now it looks like this 
![image](https://user-images.githubusercontent.com/3360588/154307887-cab98e94-eb5c-4e35-9645-ac1cd4018e1f.png)


```suggestion
Keep this in mind if you’re modifying Carbon or creating a custom component:

- The inline code text is implemented as a button so its text can be copied
- Single line snippets take an additional tabstop to support arrow key scrolling
```